health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/UploadDataMapping.js

```
@@ -30,6 +33,42 @@ const reducer = (state, action) => {
         arrayBuffer: action.payload?.arrayBuffer,
         schemas: action?.schemas,
       };
+    case "ADD_DATA":
+      const updatedData = [...state.data, action.payload];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)